e2e/playwright/point-click.spec.ts

```diff
@@ -1,55 +1,359 @@
       headerArguments: { Selection: '1 face', Distance: '' },
       highlightedHeaderArg: 'distance',
       commandName: 'Extrude',
     })
-    await app.progressCmdBar()
+    await cmdBar.progressCmdBar()
 
     const expectString = 'const extrude001 = extrude(5, sketch001)'
-    await app.expectEditor.not.toContain(expectString)
+    await editor.expectEditor.not.toContain(expectString)
 
-    await app.expectCmdBarToBe({
+    await cmdBar.expectState({
       stage: 'review',
       headerArguments: { Selection: '1 face', Distance: '5' },
       commandName: 'Extrude',
     })
-    await app.progressCmdBar()
+    await cmdBar.progressCmdBar()
+
+    await editor.expectEditor.toContain(expectString)
+  })
+})
+
+test.describe('verify sketch on chamfer works', () => {
+  const _sketchOnAChamfer =
+    (
+      app: AuthenticatedApp,
+      editor: EditorFixture,
+      toolbar: ToolbarFixture,
+      scene: SceneFixture
+    ) =>
+    async ({
+      clickCoords,
+      cameraPos,
+      cameraTarget,
+      beforeChamferSnippet,
+      afterChamferSelectSnippet,
+      afterRectangle1stClickSnippet,
+      afterRectangle2ndClickSnippet,
+    }: {
+      clickCoords: { x: number; y: number }
+      cameraPos: { x: number; y: number; z: number }
+      cameraTarget: { x: number; y: number; z: number }
+      beforeChamferSnippet: string
+      afterChamferSelectSnippet: string
+      afterRectangle1stClickSnippet: string
+      afterRectangle2ndClickSnippet: string
+    }) => {
+      const [clickChamfer] = scene.makeMouseHelpers(
+        clickCoords.x,
+        clickCoords.y
+      )
+      const [rectangle1stClick] = scene.makeMouseHelpers(573, 149)
+      const [rectangle2ndClick, rectangle2ndMove] = scene.makeMouseHelpers(
+        598,
+        380,
+        { steps: 5 }
+      )
+
+      await scene.moveCameraTo(cameraPos, cameraTarget)
+
+      await test.step('check chamfer selection changes cursor positon', async () => {
+        await expect(async () => {
+          // sometimes initial click doesn't register
+          await clickChamfer()
+          await editor.expectActiveLinesToBe([beforeChamferSnippet.slice(-5)])
+        }).toPass({ timeout: 40_000, intervals: [500] })
+      })
+
+      await test.step('starting a new and selecting a chamfer should animate to the new sketch and possible break up the initial chamfer if it had one than more tag', async () => {
+        await toolbar.startSketchPlaneSelection()
+        await clickChamfer()
+        // timeout wait for engine animation is unavoidable
+        await app.page.waitForTimeout(600)
+        await editor.expectEditor.toContain(afterChamferSelectSnippet)
+      })
+      await test.step('make sure a basic sketch can be added', async () => {
+        await toolbar.rectangleBtn.click()
+        await rectangle1stClick()
+        await editor.expectEditor.toContain(afterRectangle1stClickSnippet)
+        await rectangle2ndMove({
+          pixelDiff: 50,
+        })
```

Comment:
I integrated an option pixelDiff into the move and click helpers, i.e do the click and wait until there's a pixel diff.